crates/topos-tce-broadcast/src/double_echo/broadcast_state.rs

```diff
@@ -230,13 +230,20 @@ impl BroadcastState {
 
     fn reached_delivery_threshold(&self) -> bool {
         // If reached the delivery threshold, I can deliver
-        match self
+        let delivery_threshold = match self
             .subscriptions_view
             .network_size
             .checked_sub(self.subscriptions_view.ready.len())
         {
             Some(consumed) => consumed >= self.delivery_threshold,
             None => false,
-        }
+        };
+
+        debug!(
+            "📝 Certificate {} reached Delivery threshold: {}",
+            &self.certificate.id, delivery_threshold
+        );
```

Comment:
This one will print out often and redundant with logs in `update_status` no? Maybe to remove or to put it as `trace`?

Reply:
It matches what we have for `echo` and `ready` threshold, when the certificate is delivered, even if we have more message coming, they're ignored